celesta-sql/src/main/javacc/CelestaParser.jj

```diff
@@ -118,6 +118,11 @@ PARSER_END(CelestaParser)
     <K_GRAIN_VERSION: "VERSION">
 }
 
+<SCHEMA_OPTIONS_STATE> TOKEN:
```

Comment:
I would prefer not to create new lexer state here!

Better just rename `TABLE_OPTIONS_STATE` to `OPTIONS_STATE` and reuse it both for table and grain options. It all will be shorter